spec/rubocop/cop/rails/uniq_before_pluck_spec.rb

```diff
@@ -3,55 +3,83 @@
 
 require 'spec_helper'
 
-describe RuboCop::Cop::Rails::UniqBeforePluck do
-  let(:corrected) { 'Model.uniq.pluck(:id)' }
-  subject(:cop) { described_class.new }
+describe RuboCop::Cop::Rails::UniqBeforePluck, :config do
+  subject(:cop) { described_class.new(config) }
 
-  shared_examples_for 'UniqBeforePluck cop' do |source, action|
+  shared_examples_for 'UniqBeforePluck cop' do |method, source, action|
     if action == :correct
```

Comment:
I'm leaning towards restoring auto-correct, but making it disabled by default. Even with the conservative mode, I think there is a possibility of false positives because the cop only checks that `pluck` is called on a constant but it cannot verify that the constant refers to a model class.